src/docs/asciidoc/variables.adoc

```diff
@@ -18,4 +18,7 @@ endif::[]
 ifndef::contact-url[]
 :contact-url: {main-site-url}/contact/
 endif::[]
+ifndef::cometd-url[]
+:cometd-url: https://cometd.org/
```

Comment:
Ok to use variables for URLs but not necessarily for external URLs. Variables for URLs are meant for ZetaPush URLs in order to build documentation for several platforms. If an external URL changes, we need to update the documentation not only the URL

Reply:
I think we should use variables to preserve coherence between link to different external tools we use.